js/bundle/tests/decoder_test.js

```diff
@@ -0,0 +1,52 @@
+const wbn = require('../lib/wbn');
+
+describe('Bundle', () => {
+  const bundleBuffer = (() => {
+    const builder = new wbn.BundleBuilder('https://example.com/');
+    builder.setManifestURL('https://example.com/manifest.json');
```

Comment:
We should have no-manifest wbn test case.

Reply:
The pre-generated wbn doesn't have a manifest.